### ISSUE TITLE
eclipse -color -theme

### ISSUE THREAD

**Issue by mtyler1961** [eclipse-color-theme/eclipse-color-theme#264]
I was wondering does this work on the Ibm RDI RPG editor. I tried using it and it used up a lot of memory on my system and failed to work. I'm not sure if it's compatible with that editor.

**Comment by ryanbuening** [comment]
@mtyler1961 did you ever find a good dark Eclipse theme for RDI?

**Comment by mtyler1961** [comment]
No, I never got a response back. So, I'm assuming there isn't one available

**Comment by ryanbuening** [comment]
@mtyler1961 FYI - I did come across a built-in "dark" theme in RDi. If you go to Preferences > LPEX Editor > Appearance, change the **Palette** to **black**.

**Comment by mtyler1961** [comment]
@ryanbuening , yes I've been using that one for a while now. I was just looking for some more unique stuff.  Thanks
